subsystem/kubeapi_test.go

```diff
@@ -1620,6 +1622,11 @@ var _ = Describe("[kube-api]cluster installation", func() {
 			return infraEnv.Status.InfraEnvDebugInfo.EventsURL
 		}, "30s", "10s").Should(MatchRegexp(fmt.Sprintf("/v2/events.*infra_env_id=%s", infraEnv.ID.String())))
 
+		infraEnvCr := getInfraEnvCRD(ctx, kubeClient, infraEnvKey)
+		events, err := testEventUrl(infraEnvCr.Status.InfraEnvDebugInfo.EventsURL)
+		Expect(err).To(BeNil())
+		Expect(events).NotTo(BeEmpty())
```

Comment:
I haven't cherry-picked this yet, but is it possible to check the Payload len so we can know we have events count > 0?

Reply:
Will this always be true in all tests? If yes, we can this check in the `testEventUrl` func.

Reply:
Added the check in the func. Let see if tests pass